src/nvim/option_defs.h

```diff
@@ -567,21 +567,6 @@ EXTERN char_u   *p_titlestring; /* 'titlestring' */
 EXTERN char_u   *p_tsr;         /* 'thesaurus' */
 EXTERN int p_ttimeout;          /* 'ttimeout' */
 EXTERN long p_ttm;              /* 'ttimeoutlen' */
-EXTERN long p_ttyscroll;        /* 'ttyscroll' */
-#if defined(FEAT_MOUSE) && defined(UNIX)
-EXTERN char_u   *p_ttym;        /* 'ttymouse' */
```

Comment:
@tarruda can you confirm that we're not going to use `'ttymouse'` and `'ttyscroll'` in the future?